### PR TITLE
fix a few typos

### PR DESCRIPTION
**Pull request by benmarwick**

### CODE REVIEW

File: vignettes/manifesto.Rmd [hunk 1]
```diff
@@ -38,7 +38,7 @@ Where possible, the primary object for your API should be a tidy tibble:
 
 Tidy tibbles have the advantage of being an extremely common data structure so there are many tools that already work with them (ggplot2, dplyr, tidyr, all the modelling functions, ...). 
 
-If a tibble is not appropriate, because you're not working with variables and observations, use with a small number of well defined data structures. Some examples:
+If a tibble is not appropriate, it's because you're not working with variables and observations, and instead you should use a small number of well defined data structures. Some examples:
```
Comment (benmarwick) [inline vignettes/manifesto.Rmd]: A bit ambiguous here, is this what you're trying to say?

Comment (hadley) [inline vignettes/manifesto.Rmd]: I think eliminating the "with" best captures my meaning. But this vignette is still quite rough so I don't know if it's worthwhile to give a fine reading yet.

**Comment by hadley** [comment]
I just took another pass through the whole doc which unfortunately made most of your changes no longer apply.

**Comment by benmarwick** [comment]
No worries. Great package, very handy, thanks!

**Pull request closed without merging**